test/regression/RegressionTest.cc

```diff
@@ -92,7 +93,15 @@ void RegressionTest::run(const char* source, const char* triple,
       createPortAllocator();
 
   // Create a branch predictor for a pipelined core
-  simeng::GenericPredictor predictor = simeng::GenericPredictor();
+  simeng::PerceptronPredictor predictor = simeng::PerceptronPredictor();
```

Comment:
This line of code seems redundant

Reply:
Yep, very much so.  This was a remnant of the process of fixing the YAML config issue I had